core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -353,7 +387,7 @@ protected Path createVolumeDirectory(boolean temporary) {
     }
 
     protected void configure() {
-
+        getDockerImageName();
```

Comment:
`configure` is user-overridable method, and also there is no need to trigger it here